infra/app/app-config/main.tf

```diff
@@ -21,8 +21,6 @@ locals {
 
   has_incident_management_service = false
 
-  feature_flags = ["foo", "bar"]
-
```

Comment:
not really relevant to this PR but i decided split this out into a separate file